docs/source/features/ihv-integration/openvino.md

```diff
@@ -6,28 +6,45 @@ platforms.
 
 Read more at: [Intel® Distribution of OpenVINO™ Toolkit](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html)
 
+The `nncf` package (Neural Network Compression Framework) is used for model compression and quantization. It is required for workflows involving post-training quantization or other advanced optimization techniques in OpenVINO.
+
+For Generative AI models, install Optimum Intel® from [Optimum Intel® Installation Instructions](https://huggingface.co/docs/optimum/main/en/intel/installation)
 
 ## Prerequisites
-Note: OpenVINO version in Olive: 2023.2.0
+
+Note: OpenVINO version in Olive: 2025.1.0
```

Comment:
We have a lot OpenVINO passes that requiring 2025, do we want to update this as well? https://github.com/microsoft/Olive/blob/main/olive/olive_config.json#L587

Reply:
Nm, i saw you updated this already.